code/modules/SCP/SCPs/SCP-173.dm

```diff
@@ -25,11 +26,26 @@ GLOBAL_LIST_EMPTY(scp173s)
 	..()
 	GLOB.scp173s += src
 	verbs += /mob/living/proc/ventcrawl
+	add_language(LANGUAGE_GALCOM, 1) // it's a fucking magical statue
```

Comment:
I believe I changed this language identifier. Can you check that out, please? Unless I didn't, then it's fine.

Reply:
It's a define.

`#define LANGUAGE_GALCOM "English"`